asapdiscovery-dataviz/asapdiscovery/dataviz/html_viz.py

```diff
@@ -571,7 +573,12 @@ def make_logoplot_input(self, resi) -> dict:
             ]
 
         logoplot_base64s_dict = {}
-        with tempfile.TemporaryDirectory() as tmpdirname:
+
+        # hide a shockingly large number of prints from inside logomaker
+        with tempfile.TemporaryDirectory() as tmpdirname, HiddenPrint() as _:
+            import matplotlib
+
+            matplotlib.use("agg")
```

Comment:
I'm assuming this prevents some annoying logging from happening in the `logomaker` call below, but just want to make sure this isn't extra code

Reply:
Yep, spot on, stops matplotlib trying to start a GUI and giving a bunch of warnings.